subt_example/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ add_executable(teleop_node src/teleop_node.cc)
 add_dependencies(teleop_node ${catkin_EXPORTED_TARGETS})
 target_link_libraries(teleop_node ${catkin_LIBRARIES})
 
-add_subdirectory(src/truth_controller)
+#add_subdirectory(src/truth_controller)
```

Comment:
I also had to do this. I think you can just remove this line completely.

Reply:
Removed in [d7abecb](https://github.com/osrf/subt/pull/640/commits/d7abecb65bc2e8598b1073d1b6041b3a753ea177).